react/Viewer/index.jsx

```diff
@@ -149,6 +151,19 @@ const ViewerInformationsWrapper = ({
   validForPanel,
   toolbarRef
 }) => {
+  const theme = useTheme()
+  const sidebar = document.querySelector('[class*="sidebar"]')
+
+  useSetFlagshipUI(
+    {
+      bottomBackground: theme.palette.background.paper,
+      bottomTheme: 'dark'
+    },
+    {
+      bottomBackground: theme.palette.background[sidebar ? 'default' : 'paper']
```

Comment:
nit: I wonder if we should have set a default value inside useSetFlagshipUI

I'm not sure. Maybe you already define not to have some default values.

Reply:
We do have some depending on which screen you're using (appscreen or homescreen). Only new values override past values.